<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgurAnonyUploader</class>
 <widget class="QWidget" name="ImgurAnonyUploader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="1">
    <widget class="QLineEdit" name="appCredits">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="userCredits">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="userResetLabel">
     <property name="text">
      <string>Odnowienie kredytów</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateTimeEdit" name="userReset">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="userCreditsLabel">
     <property name="text">
      <string>Kredyty na użytkownika</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="appCreditsLabel">
     <property name="text">
      <string>Kredyty na aplikację</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
